docs/user-guide/options.md

```diff
@@ -212,6 +212,12 @@ CLI flags: `--report-needless-disables, --rd`
 
 Report `stylelint-disable` comments that don't actually match any lints that need to be disabled.
 
+## `validate`
+
+CLI flags: [`--validate, --no-validate`](cli.md#--validate---no-validate)
+
+Force enable/disable the validation of the rules' options.
+
```

Comment:
[suggestion] This is not mandatory, but having an example might be helpful. E.g.

````md
For example, your CI might be faster by skipping the validation:

```js
config = {
  validate: process.env['CI'] !== 'true'
};
```
````

I'm unsure, but editors like VS Code might have a similar environment variable.